recipes/geeadd/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Since you are the maintainer of https://github.com/samapriya/gee_asset_manager_addon , could you please make sure that the license file is packaged as part of the sdist, so that vendoring it here is not needed?

Reply:
License is now part of sdist in v0.4.5

Reply:
@nehaljwani It is ready for your review. Thanks!